src/content/docs/showcase/widgets.mdx

```diff
@@ -68,4 +68,4 @@ https://github.com/ratatui-org/ratatui-website/issues/249 for information about
 
 ## Third Party Widgets
 
-See the next page for a showcase of [third party widgets](../widgets/third-party-widgets).
+See the next page for a showcase of [third party widgets](./third-party-widgets).
```

Comment:
Looks like the link is still broken, it misses `..`. Strange because it worked locally. Let me open another PR.

Reply:
Done: https://github.com/ratatui-org/ratatui-website/pull/570